### ISSUE TITLE
BUG:the current binging index is not right when use whereIn

### ISSUE THREAD

**Issue by zhenyangze** [GeneaLabs/laravel-model-caching#241]
**Describe the bug**
when use whereIn, the current binging index would not right 

**Eloquent Query**
Please provide the complete eloquent query that caused the bug, for example:
and i expect:
```php
$model->whereIn('id', [1,2,3,4,5])->where('province', 919)->where('subject', 0)->whereBetween('id', [1, 99999])->get()
```

**Stack Trace**
when i use search like above query, the cacheKey will return 
```
subject-id_in_1_2_3_4_5-province_=_919-subject_=_0-id_between_4_5
```
and i hope:
```php
subject-id_in_1_2_3_4_5-province_=_919-subject_=_0-id_between_1_99999
```

**Environment**
 - PHP: [e.g. 7.1.0]
 - OS: [e.g. Ubuntu 18.04]
 - Laravel: [e.g. 5.6.15]
 - Model Caching: [e.g. 0.2.61]

**Additional context**
in file: `vendor/genealabs/laravel-model-caching/src/CacheKey.php`
the getInClauses and getNotInClauses:
```php
$this->currentBinding ++;
```
may be changed

**Comment by mikebronner** [comment]
@zhenyangze Thanks for reporting this! I will look into it soon. :)

**Comment by tmishutin** [comment]
Can confirm. Have the same issue.

`$files = File::where(function($q) use ($variableIn, $project) {
    $q->where('project_id', $project->id)->orWhereIn('variable', $variableIn);
});`

**Comment by mikebronner** [comment]
Likely related to #225

**Comment by mikebronner** [comment]
@zhenyangze Please provide the exact versions of Laravel and the plugin that are installed. Check `composer.lock` to get that information.

**Comment by mikebronner** [comment]
This is now fixed as of `0.4.15`.
